src/utils.js

```diff
@@ -12,6 +12,43 @@ import { PACKAGE_TYPES, LOCAL_PROTOCOLS } from 'const';
 /* global nodeRequire, localesRoot */
 
 const SOURCE_MAP_RE = /\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/;
+// Firefox's version format is laxer than Chrome's, it accepts:
+// https://developer.mozilla.org/en-US/docs/Toolkit_version_format
```

Comment:
This page has been moved and the current url is https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/version/format

Reply:
updated